src/components/section-navigation/_section-navigation.scss

```diff
@@ -9,6 +9,16 @@
     padding: 0;
   }
 
+  &__header {
+    font-size: 1rem;
+    margin: 0.75rem 0 0.6rem;
+  }
+
+  &--sub {
+    padding-bottom: 0.5rem;
+    padding-top: 0.5rem;
+  }
+
   &__link {
```

Comment:
I think we might not need to create these two classes as we could use these ones: 
ons-u-fs-r--b ons-u-mb-s